apps/omg_watcher_info/test/omg_watcher_info/db/eth_event_test.exs

```diff
@@ -349,28 +383,163 @@ defmodule OMG.WatcherInfo.DB.EthEventTest do
+
+      _ = insert(:ethevent, event_type: :deposit, txoutputs: [deposit_output_1])
+      _ = insert(:ethevent, event_type: :deposit, txoutputs: [deposit_output_2])
+
+      %{data: [event_1]} = DB.EthEvent.get_events(@default_paginator, nil, owner_1)
+      %{data: [event_2]} = DB.EthEvent.get_events(@default_paginator, nil, owner_2)
+
+      assert event_1 |> Map.get(:txoutputs) |> Enum.at(0) |> Map.get(:owner) == owner_1
+      assert event_2 |> Map.get(:txoutputs) |> Enum.at(0) |> Map.get(:owner) == owner_2
+    end
+
+    @tag fixtures: [:phoenix_ecto_sandbox]
+    test "includes ethevents with multiple outputs if at least one is owned by the given address" do
+      owner_1 = <<1::160>>
+      owner_2 = <<2::160>>
+
+      txo_1 = build(:txoutput, %{owner: owner_1})
+      txo_2 = build(:txoutput, %{owner: owner_2})
+      txo_3 = build(:txoutput, %{owner: owner_2})
+
+      inserted_event = insert(:ethevent, txoutputs: [txo_1, txo_2])
+      _ = insert(:ethevent, txoutputs: [txo_3])
+
+      %{data: [returned_event]} = DB.EthEvent.get_events(@default_paginator, nil, owner_1)
+
+      assert inserted_event.root_chain_txhash == returned_event.root_chain_txhash
+    end
+
+    @tag fixtures: [:phoenix_ecto_sandbox]
+    test "filters Ethereum events by type if given an event_type argument" do
+      _ = insert(:ethevent, event_type: :deposit)
+      _ = insert(:ethevent, event_type: :standard_exit)
+      _ = insert(:ethevent, event_type: :standard_exit)
+
+      %{data: [deposit]} = DB.EthEvent.get_events(@default_paginator, :deposit)
+
+      %{data: [se_1, se_2]} = DB.EthEvent.get_events(@default_paginator, :standard_exit)
+
+      assert Map.get(deposit, :event_type) == :deposit
+      assert Map.get(se_1, :event_type) == :standard_exit
+      assert Map.get(se_2, :event_type) == :standard_exit
+    end
+
+    @tag fixtures: [:phoenix_ecto_sandbox]
+    test "returns Ethereum events sorted by descending eth_height" do
+      _ = insert(:ethevent, eth_height: 1)
+      _ = insert(:ethevent, eth_height: 3)
+      _ = insert(:ethevent, eth_height: 2)
+
+      results = DB.EthEvent.get_events(@default_paginator, :deposit)
+
+      assert results.data |> Enum.at(0) |> Map.get(:eth_height) == 3
+      assert results.data |> Enum.at(1) |> Map.get(:eth_height) == 2
+      assert results.data |> Enum.at(2) |> Map.get(:eth_height) == 1
+    end
+
+    @tag fixtures: [:phoenix_ecto_sandbox]
+    test "pagination - correctly paginates responses" do
+      _ = insert(:ethevent)
+      _ = insert(:ethevent)
+      _ = insert(:ethevent)
+
+      paginator_1 = %Paginator{
+        data: [],
+        data_paging: %{
+          limit: 2,
+          page: 1
+        }
+      }
+
+      paginator_2 = %Paginator{
+        data: [],
+        data_paging: %{
+          limit: 2,
+          page: 2
+        }
+      }
+
+      %{data: data_page_1} = DB.EthEvent.get_events(paginator_1)
+      %{data: data_page_2} = DB.EthEvent.get_events(paginator_2)
+
```

Comment:
```suggestion
      %{data: data_page_1} = EthEvent.get_events(paginator_1)
      %{data: data_page_2} = EthEvent.get_events(paginator_2)

```

Reply:
Unless you have a strong objection, I'd prefer to keep `alias OMG.WatcherInfo.DB` because: 
 - `DB.TxOutput` is also being used in the test module. 
 -  This `alias` pattern is consistent with the rest of the the`watcher_info/db` test suite.